src/common/Emoji.tsx

```diff
@@ -18,43 +18,17 @@ const Emoji = ({
       ])}
       onClick={onClick}
     >
-      {emojiByName(emojiName)}
+      {EmojiName[emojiName]}
     </div>
   );
 };
 
 export const EmojiName = {
```

Comment:
```suggestion
export const EmojiByName = {
```

How about this for new object?

Reply:
Still need the exporting?

Reply:
Thanks. Addressed in https://github.com/mobu-of-the-world/mobu/pull/481/commits/abe0df5e49ab8538f5730b1f394b2b8ba24554bc